source/extensions/filters/network/zookeeper_proxy/zookeeper_utils.cc

```diff
@@ -0,0 +1,41 @@
+#include "extensions/filters/network/zookeeper_proxy/zookeeper_utils.h"
+
+namespace Envoy {
+namespace Extensions {
+namespace NetworkFilters {
+namespace ZooKeeperProxy {
+
+void BufferHelper::peekInt32(Buffer::Instance& buffer, uint64_t& offset, int32_t& val) {
+  val = buffer.peekBEInt<int32_t>(offset);
+  offset += sizeof(int32_t);
+}
+
+void BufferHelper::peekInt64(Buffer::Instance& buffer, uint64_t& offset, int64_t& val) {
+  val = buffer.peekBEInt<int64_t>(offset);
+  offset += sizeof(int64_t);
+}
+
+void BufferHelper::peekBool(Buffer::Instance& buffer, uint64_t& offset, bool& val) {
+  char byte = buffer.peekInt<char, ByteOrder::Host, 1>(offset);
```

Comment:
My original question regarding this was about this one use case that did *not* use BE but uses Host ordering instead. Thinking about it some more I think this might be because it's a single byte so it doesn't matter? I'd still probably use BE here just for the sake of consistency.

Reply:
Oh, right — doesn't matter here.